exploring.rst

```diff
@@ -3,8 +3,9 @@
 Exploring CPython's Internals
 =============================
 
-This is a quick guide for who want to know about CPython's internals,
-and provide a summary about the source code structure and references.
+This is a quick guide for people who interested in learning more about
```

Comment:
@grapherd one small change to your corrected wording

...who are interested...